### PR TITLE
01 oct

### PR DESCRIPTION
**Pull request by SVARusu**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,59 +6,45 @@
...
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.8/css/all.css">
     <link rel="stylesheet" href="css/style.css">
     <title>Document</title>
 </head>
 
 <body>
-    <div class="container register">
-        <div class="card bg-light">
+    <div class="container login">
+        <div class="card bg-light" style="min-width: 500px;">
```
Comment (svpgaicean) [inline index.html]: don't use inline styles

File: js/db.ts [hunk 2]
```diff
@@ -151,11 +145,8 @@ class DB {
 
     printTodos() {
         return new Promise((resolve, reject) => {
-            // console.log(notesList);
-            console.log(this.db);
             let db;
```
Comment (svpgaicean) [inline js/db.ts]: declare 'db' where it's used -> line 151

File: js/db.ts [hunk 3]
```diff
@@ -164,25 +155,113 @@ class DB {
...
                 }
 
             }
         })
     }
+
+
+    printCompletedTodos() {
+        return new Promise((resolve, reject) => {
+            let db;
```
Comment (svpgaicean) [inline js/db.ts]: declare 'db' where it's used -> line 184

File: js/notespage.ts [hunk 4]
```diff
@@ -26,34 +39,69 @@ function init() {
...
         });
-}
\ No newline at end of file
+
+    db.printCompletedTodos()
+        .then((completedTodos: any) => {
+            while (completedNotesList.firstChild) {
+                completedNotesList.removeChild(completedNotesList.firstChild as ChildNode);
+            }
+            for (let i = 0; i < completedTodos.length; i++) {
```
[the remark this answers is not in the document]
Comment (SVARusu) [reply js/notespage.ts]: might want to look at those fors again

File: js/db.ts [hunk 5]
```diff
@@ -164,25 +155,113 @@ class DB {
...
+            }
+
+
+        }
+    }
 }
 
-export { DB };
\ No newline at end of file
+export { DB, cursorValue };
```
[the remark this answers is not in the document]
Comment (SVARusu) [reply js/db.ts]: why?

File: js/app.ts [hunk 6]
```diff
@@ -1,6 +1,6 @@
 console.log("something else");
```
[the remark this answers is not in the document]
Comment (SVARusu) [reply js/app.ts]: it's something

**Pull request merged**